src/styles/public/patterns.scss

```diff
@@ -294,22 +295,22 @@ html body.wp-admin .editor-styles-wrapper {
 
 	&.has-accent-stroke {
 
-		img {
+		.wp-block-media-text__media img {
```

Comment:
.wp-block-media-text {
   &__media img { <- this will allow the BEM style structure to be used without suppressing selector-class-pattern. That said, I don't like having to do this and I feel it is a CSS lint that should be omitted from SCSS rules.